src/plugins/shared_ux/public/components/empty_state/no_data_views/no_data_views.test.tsx

```diff
@@ -12,8 +12,12 @@ import { ReactWrapper } from 'enzyme';
 import { mountWithIntl } from '@kbn/test-jest-helpers';
 import { EuiButton } from '@elastic/eui';
 
-import { ServicesProvider, SharedUXServices } from '../../../services';
-import { servicesFactory } from '../../../services/mocks';
+import {
```

Comment:
nit: in platform deployment management we took the habit of importing all external static code from a top level `shared_imports.ts` file. It had several benefits:

- Reducing the length of the relative path inside the plugin (less "../../../")
- Being able to see in a single place all the dependencies to other plugins
- Being able to swap an import without changing the consuming parts (not frequent but it occurred sometimes)

https://github.com/elastic/kibana/blob/main/x-pack/plugins/index_management/public/shared_imports.ts
https://github.com/elastic/kibana/blob/main/x-pack/plugins/ingest_pipelines/public/shared_imports.ts